stl/inc/atomic

```diff
@@ -377,23 +365,23 @@ struct _Atomic_padded {
 #else // ^^^ don't break ABI / break ABI vvv
 template <class _Ty>
 struct _Atomic_storage_traits { // properties for how _Ty is stored in an atomic
-    static constexpr size_t _Storage_size = sizeof(_Ty) == 1  ? 1
-                                          : sizeof(_Ty) == 2  ? 2
-                                          : sizeof(_Ty) <= 4  ? 4
-                                          : sizeof(_Ty) <= 8  ? 8
+    static constexpr size_t _Storage_size = sizeof(_Ty) == 1 ? 1
+                                          : sizeof(_Ty) == 2 ? 2
+                                          : sizeof(_Ty) <= 4 ? 4
+                                          : sizeof(_Ty) <= 8 ? 8
 #if defined(_M_X64) || defined(_M_ARM64) || defined(_M_ARM64EC)
                                           : sizeof(_Ty) <= 16 ? 16
 #endif // 64 bits
                                                               : sizeof(_Ty);
 
     static constexpr size_t _Padding_size = _Storage_size - sizeof(_Ty);
-    static constexpr bool _Uses_padding   = _Padding_size != 0;

```

Comment:
These hunks are stray, but CI is not complaining.

Reply:
I'm pretty sure that this is clang-format's doing, in response to the preprocessor directives being removed above - I've noticed this kind of behavior elsewhere. Should be nothing to worry about, and Code Format Validation is happy as you mentioned.